src/lib/Tisch/Internal/Table.hs

```diff
@@ -13,6 +14,7 @@
 {-# LANGUAGE RankNTypes #-}
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE TemplateHaskell #-}
```

Comment:
Sure, but it involves writing (or copying from `-ddump-splices`) a bunch of boilerplate which could go out of date,